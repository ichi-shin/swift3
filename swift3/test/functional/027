#!/bin/bash

# Upload Part

. ./common

S3USER=tester

_s3_put /bucket

upload_id=$(_s3_post /bucket/object?uploads | \
    _xpath '/InitiateMultipartUploadResult/UploadId/text()')
_s3_put /bucket/object?uploadId=${upload_id}\&partNumber=10000 -T /dev/null -D - | \
    _filter_curl

_sw_get /bucket+segments | _filter_curl | sed -e "s/${upload_id}/UPLOAD_ID/g"
