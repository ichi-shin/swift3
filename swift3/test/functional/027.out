QA output created by 027
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X POST http://SWIFT_HOST/bucket/object?uploads... 200
> s3curl --id tester -- -X PUT -T /dev/null -D - http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=10000... 200
HTTP/1.1 100 Continue

HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: text/html; charset=UTF-8
Date: DATE
ETag: "ETAG"
Last-Modified: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

> curl -X GET http://SWIFT_HOST/v1/TENANT/bucket+segments... 200
object/UPLOAD_ID
object/UPLOAD_ID/10000
